mri/generators/base.py

```diff
@@ -0,0 +1,94 @@
+"""
+Base Class for KspaceGenerators.
```

Comment:
```suggestion
Base Class for k-space generators.
```

Note that KspaceGenerators is the base class right?